chains/manager.go

```diff
@@ -235,6 +234,9 @@ type ManagerConfig struct {
 
 	ApricotPhase4Time            time.Time
 	ApricotPhase4MinPChainHeight uint64
+	CortinaTime                  time.Time
+	CortinaXChainStopVertexID    ids.ID
+	DurangoTime                  time.Time
```

Comment:
May be nicer to pass the whole upgrade struct through, but kept this for now since it followed the previous style.